SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -71,12 +78,12 @@ protected override SonarQubeIssue[] ParseResponse(string response)
                 .ToLookup(c => c.Key, c => c.Path); // Using a Lookup because it does not throw, unlike the Dictionary
         }
 
-        private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue, ILookup<string, string> componentKeyPathLookup) =>
-            new SonarQubeIssue(ComputePath(issue, componentKeyPathLookup), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
+        private SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
+            new SonarQubeIssue(ComputePath(issue.Component), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
                 GetRuleKey(issue.CompositeRuleKey), issue.Status == "RESOLVED", ToIssueFlows(issue.Flows));
 
-        private static string ComputePath(ServerIssue issue, ILookup<string, string> componentKeyPathLookup) =>
-            componentKeyPathLookup[issue.Component].FirstOrDefault() ?? string.Empty;
+        private string ComputePath(string component) =>
```

Comment:
Modified the method to expect the component string so we can use it for `ServerIssueLocation` instances too.